test/test_quantized.py

```diff
@@ -384,6 +384,39 @@ def test_qlinear_relu(self):
         # Assert equal
         np.testing.assert_equal(Y_q_ref2.int_repr().numpy(), Y_q.int_repr().numpy())
 
+    """Tests the correctness of the quantized::fbgemm_linear_unpack op."""
+    def test_fbgemm_linear_unpack(self):
+        qlinear_prepack = torch.ops.quantized.fbgemm_linear_prepack
+        qlinear_unpack = torch.ops.quantized.fbgemm_linear_unpack
+
+        input_channels = 16
+        output_channels = 8
+
+        W_scale = 0.4
+        # W_zp is the zero point for int8 quantization.
+        W_zp = 2
+        W_value_min = -128
+        W_value_max = 127
+        W_q0 = np.round(
+            # np.random.rand(output_channels, input_channels)
+            np.ones((output_channels, input_channels))
+            * (W_value_max - W_value_min)
+            + W_value_min
+        ).astype(np.int8)
+
+        W = torch.from_numpy(_dequantize(W_q0, W_scale, W_zp)).to(dtype=torch.float)
+        W_q = W.quantize_linear(scale=W_scale, zero_point=W_zp, dtype=torch.qint8)
```

Comment:
could you use `torch.quantize_linear(...)` here

Reply:
Will do.